content/product-engineering/planning.md

```diff
@@ -0,0 +1,140 @@
+| ------------- | -------------------------------------------- |
+| 🟢 On track   | 90-100% confidence we will achieve the goal. |
+| 😬 At risk    | 75-89% confidence we will achieve the goal.  |
+| ⛔️ Off track | <75% confidence we will achieve the goal.    |
+| 🚀 Done       | We have achieved the goal!                   |
+| ☠️ Cancelled  | We are no longer pursuing this goal.         |
+
+### GitHub project tracker
+
+The [GitHub project tracker](https://github.com/orgs/sourcegraph/projects/214) is **the source of truth** for key results in product and engineering. There are a few tabs:
+
+- The **Product and Engineering** tab tracks the results that the product and engineering VPs and directors review every week at the [VP team sync](#vp-team-sync).
+- Each org (Cloud, Enablement, Code graph) has a tab that tracks the results for that org. Org directors review this dashboard each week with their team.
+- The **Engineering** tab tracks results that are relevant only to engineering or engineering leadership. This is reviewed every week in the [VPE team sync](#todo-link).
+- The **All** tab simply shows all items in the tracker for reference.
+
+Expectations:
+
+- Each tab on this tracker is reviewed and updated on a weekly basis with the relevant teams for each tab (see above).
+- Each item in the tracker is linked to a GitHub issue.
+  - Customer names and business metrics can't be mentioned on public issues, so by default, all issues should be created in [sourcegraph/product-engineering-tracker](https://github.com/sourcegraph/product-engineering-tracker).
+- For each issue linked to from the tracker, the assignee is responsible for posting an update to the issue at the end of each week. The update should include:
+
+  ```markdown
+  # Wins
+
+  <!-- What outcomes have we accomplished since the last update? -->
+
+  # Risks
+
+  <!-- What might prevent us from making the progress we want to make? -->
+
+  # Next steps
+
+  <!-- What are the next outcomes that the team is working toward? -->
+  ```
+
+### OKR slides
+
+The OKR tracker slides are a living document that is updated once a week to communicate that current state of top level product and engineering key results for the current quarter. It only contains 1 week's worth of updates. Each slide has an owner, and that owner is responsible for updating that slide at the end of each week.
+
+The slides are reviewed and discussed each week in the [VP team sync](team.md#vp-team-sync) so we can celebrate wins, identify risks, and course correct if necessary.
+
+After the slides are reviewed, they are copied to a seprate slide deck to capture week-over-week progress.
+
+Slides:
+
+- [FY22 Q4 (current week)](https://docs.google.com/presentation/d/1DgY3k684Jn3diCe4GPPcrGt9iaD9-vyndiiJwEoELyE/edit#slide=id.gfca47eca53_0_184)
+- [FY22 Q4 (week over week history)](https://docs.google.com/presentation/d/1d36d47VFSZzQWIhdkZt3JZ3pNUk766yy8iwkFfVpTyY/edit)
+
+### Monthly team updates
```

Comment:
@nicksnyder is this something you want from all engineering teams?

I've gone back to my team to gather some initial feedback and overall group sentiment leans towards preferring Code Graph's current GitHub issues approach to providing team updates. Things people liked – process feels more lightweight, everything being in GitHub, better integration with GitHub (referencing issues/PRs). One person did point out they liked the email approach because it had a wider audience.

Regardless of the decision here, I'd love to minimize the need for duplicate updates.

@lguychard @scalabilitysolved @jjeffwarner @felixfbecker @chrispine tagging you here if wanted to comment directly

Reply:
Yes the proposal documented in this PR is for each team to send a monthly email (instead of a monthly post to a GitHub issue for that team). The reason for email is that it is easier for stakeholders outside of engineering to subscribe to (only need to subscribe to a single mailing this rather than a bunch of issues), but we could try to automate that to bridge the gap if we need to. If we used one long lived issue per team, eventually they would get really long and hard to navigate.

The goal is for this not to be duplicative with weekly tactical updates on KRs. The audience is different, level of detail is different (due to monthly vs weekly cadence)

Definitely open to feedback here!